ajax/libs/feather-icons/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "feather-icons",
+  "filename": "feather.min.js",
+  "keywords": [
+    "popular",
+    "icons"
+  ],
+  "version": "3.3.2",
```

Comment:
The latest version is `3.2.2` :) 
So the version folder needs to be revised too. You can directly remove the folder and grab the files using auto-update config like step 2 and 3. 💪